subset/mac/Dockerfile.test_mac

```diff
@@ -0,0 +1,11 @@
+FROM daq/aardvark:latest 
```

Comment:
As per other comments, filename should be something like "subset/????/Dockerfile.test_macoui"